src/main/java/it/multicoredev/carpetredcraftaddons/CarpetRedCraftExtension.java

```diff
@@ -68,6 +68,7 @@ public void onGameStarted() {
         CarpetScriptServer.registerSettingsApp(redcraftDefaultScript("treecapitator", false));
         CarpetScriptServer.registerSettingsApp(redcraftDefaultScript("villagerleash", false));
         CarpetScriptServer.registerSettingsApp(redcraftDefaultScript("waystone", false));
+        CarpetScriptServer.registerSettingsApp(redcraftDefaultScript("playerme", false));
```

Comment:
Da posizionare nel punto corretto, al fine che le regole siano disposte in ordine alfabetico